front-end/src/screens/OnChain/Motions/query.ts

```diff
@@ -3,66 +3,67 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import gql from 'graphql-tag';
+import { authorFields } from 'src/fragments/author';
 
 // for motions postType shoud be 2, postTopic should be 2
 export const QUERY_LATEST_MOTIONS = gql`
-  query LatestMotionPosts($postType: Int!, $limit: Int! = 5 ) {
```

Comment:
This changes look awful, but all I did is add the `...authorField` fragment and correct the indentation.